tests/sympc/tensor/test_mpc_tensor.py

```diff
@@ -0,0 +1,210 @@
+import pytest
+import syft as sy
+import torch
+
+from sympc.session import Session
+from sympc.tensor import MPCTensor
+
+
+@pytest.fixture
+def clients():
+    alice = sy.VirtualMachine(name="alice")
+    bob = sy.VirtualMachine(name="bob")
+    james = sy.VirtualMachine(name="james")
+
+    alice_client = alice.get_client()
+    bob_client = bob.get_client()
+    james_client = james.get_client()
+
+    return [alice_client, bob_client, james_client]
```

Comment:
@aanurraj - looking at this and @LaRiffle's comment on https://github.com/OpenMined/SyMPC/pull/22, I think it makes a lot of sense to consolidate the fixtures in some `tests/conftest.py` file.

Reply:
Yes i already made changes just need to push!

Reply:
I moved these to the `tests/conftest.py` file at https://github.com/OpenMined/SyMPC/pull/15. :smile: - what do you think?

Reply:
Ah - brilliant! Sorry... I hadn't seen that previous comment :sweat_smile:

Reply:
Ah - could you add for the moment this only for two parties - at > 2 parties we need to implement a better "truncation" function

Reply:
sure @gmuraru